src/main/java/io/anserini/rerank/lib/BM25PRFReranker.java

```diff
@@ -0,0 +1,336 @@
+package io.anserini.rerank.lib;
+
+import io.anserini.rerank.Reranker;
+import io.anserini.rerank.RerankerContext;
+import io.anserini.rerank.ScoredDocuments;
+import io.anserini.util.AnalyzerUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.index.IndexReader;
+import org.apache.lucene.index.Term;
+import org.apache.lucene.index.Terms;
+import org.apache.lucene.index.TermsEnum;
+import org.apache.lucene.search.*;
+import org.apache.lucene.search.similarities.BM25Similarity;
+import org.apache.lucene.util.BytesRef;
+
+import java.io.IOException;
+import java.util.*;
+
+import static io.anserini.index.generator.LuceneDocumentGenerator.FIELD_BODY;
+
+
+class BM25PRFSimilarity extends BM25Similarity {
+
+    BM25PRFSimilarity(float k1, float b) {
+        super(k1, b);
+    }
+
+    @Override
+    // idf is not needed in BM25PRF
+    protected float idf(long docFreq, long docCount) {
+        return 1;
+    }
+}
+
+
+public class BM25PRFReranker implements Reranker {
```

Comment:
Would `BM25PrfReranker` look better? And better match Java conventions? I'm agnostic about this... I think both look ugly-ish.

Reply:
I think `BM25PrfReranker ` is better, so I will replace `PRF`with `Prf`.